cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -305,19 +305,16 @@ def start_master(etcd):
     hookenv.status_set('maintenance',
                        'Configuring the Kubernetes master services.')
     freeze_service_cidr()
+    # TODO: Make sure below relation is handled on change not
```

Comment:
Actually it's https://github.com/kubernetes/kubernetes/issues/43461 which was incorrectly marked as "Closed". I could not find a way to reopen